wizard/WizardCreateDevice1.qml

```diff
@@ -56,6 +57,7 @@ Rectangle {
         ListElement { column1: "Ledger Nano X"; column2: "Ledger";}
         ListElement { column1: "Ledger Stax"; column2: "Ledger";}
         ListElement { column1: "Trezor Model T"; column2: "Trezor";}
+        ListElement { column1: "Trezor Model 3"; column2: "Trezor";}
```

Comment:
All marketing refers to this model as "Safe 3".

Reply:
Thanks, updated!